include/os/screen.h

```diff
@@ -9,13 +9,13 @@
 #pragma comment(lib, "user32.lib")
 #endif
 
-// OpenCV
-#include <opencv2/opencv.hpp>
+// Standard C++ includes
+#include <utility>
 
 namespace BoBRobotics {
 namespace OS {
 namespace Screen {
-cv::Size
+std::pair<int, int>
```

Comment:
How come?

Reply:
Because we don't use OpenCV elsewhere in the OS module, so this way we can drop the dependency. I did this initially to compile on the EV3 which doesn't have it installed.

Reply:
:+1: